src/screens/ContactScreen/index.js

```diff
@@ -262,7 +262,7 @@ const styles = StyleSheet.create({
     marginHorizontal: 16
   },
   subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.holytosca : COLORS.gray4,
+    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change also improves maintainability by using a reference from the `COLORS` object instead of a hardcoded color value. However, it's important to ensure that `COLORS.anon_primary` has the same color value as `COLORS.holytosca` to avoid unintended visual changes.
```diff
-    color: selectedUsers.length > 0 ? COLORS.holytosca : COLORS.gray4,
+    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->